### PR TITLE
Adds 'description' field to GCPFirewall CRD and corresponding api

### PR DESCRIPTION
**Pull request by maciejriedl**

### CODE REVIEW

File: apis/gcpfirewall/v1/types.go [hunk 1]
```diff
@@ -41,6 +41,11 @@ type GCPFirewall struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// DescriptionHello describes a GCP firewall in a human-readable way. Usually it
```
Comment (liuyuan10) [inline apis/gcpfirewall/v1/types.go:44]: DescriptionHello?

File: apis/gcpfirewall/v1/types.go [hunk 2]
```diff
@@ -41,6 +41,11 @@ type GCPFirewall struct {
...
+	// provides also an information on the producer of this GCPFirewall
```
Comment (liuyuan10) [inline apis/gcpfirewall/v1/types.go:45]: s/an information/information

File: apis/gcpfirewall/v1/types.go (hunk 1 above)
Comment (maciejriedl) [reply apis/gcpfirewall/v1/types.go]: Sorry, I must have overlooked it after doing some tests

File: apis/gcpfirewall/v1/types.go (hunk 2 above)
Comment (maciejriedl) [reply apis/gcpfirewall/v1/types.go]: fixed

**Review by liuyuan10 — APPROVED** [review]

**Pull request merged**